client/devdocs/design/blocks.jsx

```diff
@@ -129,6 +130,7 @@ export default class AppComponents extends React.Component {
 					<CalendarButton readmeFilePath="calendar-button" />
 					<CalendarPopover readmeFilePath="calendar-popover" />
 					<CommentButtons readmeFilePath="comment-button" />
+					<Checklist />
```

Comment:
Is this where you indicate the path to the readme?

Reply:
Oh, is that because there is no readme?

Reply:
That is entirely possible. I didn't make that connection until you pointed it out. :D I'll add the file path and see if it pops up!

Reply:
Just tried it, but pointing to the readme doesn't work, I think because the block itself is still located within /components, and it's looking for the readme in the /blocks folder. Will see if there's a workaround.